### PR TITLE
created top navigation

### PR DESCRIPTION
**Pull request by Chiragh-itpath**

### CODE REVIEW

File: src/ATA.Bluebook/ATA.Bluebook.Web/Common/HtmlExtensions/HtmlExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ATA.Bluebook.Web.Common.HtmlExtensions
+{
+    public static class HtmlExtensions
+    {
+        public static string IsActive(this IHtmlHelper html, params string[] controllers)
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/Common/HtmlExtensions/HtmlExtensions.cs]: Rename the Method with "GetActiveRouteClassName" and update the Related References

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_TopNavbar.cshtml [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
-            <ul class="navbar-nav me-auto">
-                <li class="nav-item"><a href="#" class="nav-link active">Home</a></li>
-                <li class="nav-item"><a href="#about" class="nav-link">About</a></li>
-                <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
-            </ul>
-        </div>
-    </div>
-</nav>
\ No newline at end of file
+﻿@using ATA.Bluebook.Web.Common.HtmlExtensions;
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_TopNavbar.cshtml]: Move it to _Import.cshtml

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_DevExtremeLayout.cshtml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+@inject IHttpContextAccessor Accessor
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_DevExtremeLayout.cshtml]: Move it to _import.cshtml and rename the variable name with "_accessor"

File: src/ATA.Bluebook/ATA.Bluebook.Web/wwwroot/css/site.css [hunk 4]
```diff
@@ -1,11 +1,24 @@
 .login-page {
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/wwwroot/css/site.css:1]: Add border-bottom to the active class for large screen

**Review by ShubhamG2908 — CHANGES_REQUESTED** [review]
Added Comments

File: src/ATA.Bluebook/ATA.Bluebook.Web/Common/HtmlExtensions/HtmlExtensions.cs (hunk 1 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/Common/HtmlExtensions/HtmlExtensions.cs]: Done

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_TopNavbar.cshtml (hunk 2 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_TopNavbar.cshtml]: Done

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_DevExtremeLayout.cshtml (hunk 3 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/Views/Shared/_DevExtremeLayout.cshtml]: Done

File: src/ATA.Bluebook/ATA.Bluebook.Web/wwwroot/css/site.css (hunk 4 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/wwwroot/css/site.css:1]: Done

**Review by ShubhamG2908 — APPROVED** [review]

**Pull request merged**